website/docs/r/ocean_gke_import.html.markdown

```diff
@@ -51,6 +51,8 @@ The following arguments are supported:
 * `max_size` - (Optional, Default: `1000`) The upper limit of instances the cluster can scale up to.
 * `min_size` - (Optional) The lower limit of instances the cluster can scale down to.
 * `desired_capacity` - (Optional) The number of instances to launch and maintain in the cluster. 
+* `whitelist` - (Optional) Instance types allowed in the Ocean cluster. Cannot be configured if `blacklist` is configured.
+* `blacklist` - (Optional) Instance types not allowed in the Ocean cluster. Cannot be configured if `whitelist` is configured.
```

Comment:
I think the `blacklist` field is not implanted in ocean_gke_import, did I miss something?

Reply:
You are correct. Blacklist is not supported in GKE. Will remove